rest-server/server/src/main/scala/com/actionml/router/http/routes/QueriesRouter.scala

```diff
@@ -40,7 +41,7 @@ class QueriesRouter(implicit inj: Injector) extends BaseRouter with AuthDirectiv
     }
   }
 
-  private def getPrediction(engineId: String, log: LoggingAdapter): Route = (get & asJson) { query =>
+  private def getPrediction(engineId: String, log: LoggingAdapter): Route = (post & asJson) { query =>
```

Comment:
Odd, how did this ever work as a `GET`???

Reply:
It was my mistake from very first branch, just fixing it